test/integration/transactions/transactions.spec.test.js

```diff
@@ -78,8 +78,16 @@ class TransactionsRunnerContext extends TestRunnerContext {
   }
 }
 
+// These tests are skipped because the driver 1) executes a ping when connecting to
+// an authenticated server and 2) command monitoring is at the connection level so
+// when the handshake fails no command started event is emitted.
+const SKIP = [
+  'AbortTransaction succeeds after handshake network error',
+  'CommitTransaction succeeds after handshake network error'
+];
+
 describe('Transactions Spec Unified Tests', function () {
-  runUnifiedSuite(loadSpecTests(path.join('transactions', 'unified')));
+  runUnifiedSuite(loadSpecTests(path.join('transactions', 'unified')), SKIP);
```

Comment:
Similar comment here as on the retryable_reads spec test file regarding moving the manual implementations from prose to this file; it would also be good to name the test cases to match the skipped descriptions